### PR TITLE
Adding Memory card game

### LINKED ISSUE #2110: [GSSOC'23] [New game] Memory Card Game
**Describe the project you want to add with tech stack**
I want to add Memory Card Game.
Tech stack: HTML, CSS, JavaScript

**Expected behavior**

- The game starts by displaying a grid of faced-down cards.
- The player can click on a card to reveal its face.
- When the player clicks on a card, the card flips over, revealing its image or symbol.
- If it's the first card the player has clicked, its value is stored for comparison.
- If it's the second card the player has clicked, its value is also stored for comparison.
- After the second card is revealed, the game checks if the two cards match.
- If the two cards match, they stay face-up, and the player earns a point.
- If the two cards do not match, they are flipped back face-down.
- The player's score is updated accordingly.
- The player continues to select pairs of cards until all cards are matched and face-up.
- The game ends and displays a message with the player's final score and a prompt to play again.

**Screenshots (optional)**

**Additional context (optional )**

### PR DESCRIPTION
**Pull request by pavitrachavda97**
# Title and Issue number 

Memory Card

Close #2110 

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@pavitrachavda97   please remove all the images or gif from your PR and use them through cloud link. Upload somewhere the images and use them from that link.

**Comment by pavitrachavda97** [comment]
@MohitGupta121
Thank you for the feedback! I have made the requested changes by removing the images from the PR. Instead, I have uploaded the images to a cloud storage service and updated the code to use cloud links. I have tested the modified code locally, and the images are loading correctly from the cloud links. The changes have been committed and pushed to the repository. Please review the updated code and let me know if any further adjustments are needed. Thank you!

File: Js-Games/memoryCard/images/img1.png
Comment (MohitGupta121) [inline Js-Games/memoryCard/images/img1.png:1]: no @pavitrachavda97 all images are still there

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by pavitrachavda97** [comment]
@MohitGupta121 
I have removed images folder now. Please review the updated code and let me know if any further adjustments are needed. 
Thank you!

**Review by MohitGupta121 — APPROVED** [review]

**Pull request merged**